libraries/tools/kotlin-main-kts/src/org/jetbrains/kotlin/mainKts/impl/ivy.kt

```diff
@@ -52,7 +51,7 @@ class IvyResolver : ExternalDependenciesResolver {
                 resolveArtifact(
                     artifactId[0], artifactId[1], artifactId[2],
                     if (artifactId.size > 3) artifactId[3] else null,
-                    if (artifactType.isNotEmpty()) artifactType else null,
+                    artifactType.ifEmpty { null },
```

Comment:
I would prefer not to have irrelevant changes in such commits. It makes history analysis more difficult. If you want to suggest such refactorings, please submit a separate PR, or at least in a separate commit.